extensions/ql-vscode/src/cli.ts

```diff
@@ -1138,23 +1138,31 @@ export class CodeQLCliServer implements Disposable {
   /**
    * Gets information about available qlpacks
    * @param additionalPacks A list of directories to search for qlpacks before searching in `searchPath`.
```

Comment:
Hmmm...this is actually technically correct, even though it might be a bit misleading. This is turning into one of the dark corners of pack resolution in the CLI.

The original idea was that people would use the `.config/codeql/config` file and hard code a `--search-path` option in the file that applies to every command anywhere on their machine. Then the workspace folders are added to the `--additional-packs` path.

While this is still true and possible, we don't recommend this way of working any more. 

So, that's a little background. And I think you're right that it is confusing and I will remove the line.